packages/account/src/providers/provider.ts

```diff
@@ -1647,4 +1688,18 @@ export default class Provider {
 
     return message;
   }
+
+  async getRelayedTransactionStatus(
```

Comment:
I couldn't find any usages of this method anywhere. Should it be deleted?

Reply:
@nedsalk Well, I reckon no one is going to use this query for now but I thought it was good to have it in place already.